integration_tests/ci/sample.profiles.yml

```diff
@@ -51,4 +51,12 @@ integration_tests:
       schema: hubspot_integration_tests_57
       threads: 8
       token: "{{ env_var('CI_DATABRICKS_DBT_TOKEN') }}"
+      type: databricks
+    databricks-sql:
```

Comment:
Should we add this to the source packages as well? Or is it out of scope.

Reply:
Added